{{cookiecutter.project_slug}}/setup.py

```diff
@@ -16,7 +16,17 @@
     # TODO: put package requirements here
 ]
 
+setup_requirements = [
+{%- if cookiecutter.use_pytest == 'y' %}
+    pytest-runner
```

Comment:
Oh yeah, that comparison isn't accomplishing anything.
Frankly adding an assertion won't do much either, because `subprocess.check_output` will raise an exception if the command return non-exit status.

You can leave out the `== 0` or add an assert, I don't mind either. =)